### PR TITLE
chore: Use Vite as main dev runner for FE

### PR DESCRIPTION
**Pull request by BrunoQuaresma**

### CODE REVIEW

File: site/package.json [hunk 1]
```diff
@@ -136,8 +134,7 @@
     "typescript": "4.8.2",
     "webpack": "5.74.0",
```
Comment (kylecarbs) [inline site/package.json:135]: Should we use Vite for production too?

Comment (BrunoQuaresma) [reply site/package.json:135]: Yes, but I have to run some experiments to check the build and build size.

**Review by kylecarbs — APPROVED** [review]

File: site/package.json [hunk 2]
```diff
@@ -136,8 +134,7 @@
...
     "webpack-bundle-analyzer": "4.6.1",
```
Comment (jsjoeio) [inline site/package.json:136]: Do we still need these?

**Review by jsjoeio — APPROVED** [review]
Long live Vite!

File: site/package.json (hunk 2 above)
Comment (BrunoQuaresma) [reply site/package.json:136]: It is kinda used to analyze the build, but I don't think we use this anywhere I will remove this after just for caution.

**Pull request merged**